### ISSUE TITLE
Periodic delay in getting kinesis video stream

### ISSUE THREAD

**Issue by utsavrai** [aws-robotics/kinesisvideo-ros1#77]
Hi,
I am experiencing a periodic delay in the frames received by the parser code provided by aws [https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/examples-renderer.html](url), I tried to change the source camera on the sender/producer side and I also tried setting different fps and resolution but the streamed video gets a constant periodic lag/pause after every 3-4 seconds. Is this issue related to producer code or the parser code? Is there any change/enhancement can be done?

**Comment by Bassem-Yousry** [comment]
did u find a solution?

**Comment by utsavrai** [comment]
No, I tried changing various things like region, network, fps, source of image etc. but it didn't change the result.
